frontend/packages/console-app/src/components/quick-starts/data/quick-start-test-data.ts

```diff
@@ -0,0 +1,16 @@
+import { exploreServerlessQuickStart } from './mocks/json/explore-serverless-quickstart';
```

Comment:
This is not needed anymore.

Reply:
I've now kept only 3 quickStarts as I am checking them in the QuickStart Catalog for 3 concurrent quickStarts. removed rest.